comp110-worksheet-6/comp110-worksheet-6.csproj

```diff
@@ -42,6 +42,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="mark.cs" />
```

Comment:
The file `mark.cs` seems to be missing from your submission -- this prevents your code from compiling or from being able to run on TravisCI